content/docs/command-reference/plots/index.md

```diff
@@ -295,9 +432,171 @@ file:///Users/usr/src/dvc_plots/index.html
 ![](/img/plots_show_confusion.svg)
 
 > A confusion matrix [template](/doc/command-reference/plots#plot-templates) is
-> predefined in DVC (found in `.dvc/plots/confusion.json`).
+> predefined in DVC.
+
+We can use `confusion_normalized` template to normalize the results:
+
+```dvc
+$ dvc plots show classes.csv --template confusion_normalized -x actual -y predicted
+file:///Users/usr/src/dvc_plots/index.html
+```
+
+![](/img/plots_show_confusion_normalized.svg)
+
+### Top-level plots
+
+#### Simple plot definition
+
+Let's get back to the `logs.csv` data:
```

Comment:
I'm not sure we want many examples if at all in the `plots` ref index page actually. Shouldn't they all be in the sub-refs? Unless a relevant example combines more than one sub-command (in this case I only see `plots show`).

Or if it's about different ways to define plots, that could be summarized into the `dvc.yaml` schema guide.

Reply:
I guess we should move the examples to `show` `diff`. The syntax is already explained in `Syntax`